tests/agent-templates.cross.auth.test.ts

```
@@ -0,0 +1,129 @@
+
+    const rows = await prisma.agentTemplate.findMany({
+      where: {
+        id: {
+          in: [jwtCreated.body.id as string, keyCreated.body.id as string],
+        },
+      },
+      orderBy: { slug: "asc" },
+      select: { id: true, ownerAccountId: true, status: true },
+    });
+    expect(rows).toHaveLength(2);
+    expect(rows.every((row) => row.ownerAccountId === ADMIN_ACCOUNT_ID)).toBe(true);
```

ESLint

[error] Replace `true` with `⏎······true,⏎····`

(prettier/prettier)